packages/blade/src/components/Input/DropdownInputTriggers/BaseDropdownInputTrigger.tsx

```diff
@@ -176,6 +167,7 @@ const _BaseDropdownInputTrigger = (
     defaultValue: props.defaultValue,
     syncInputValueWithSelection: props.syncInputValueWithSelection,
     isSelectInput: props.isSelectInput,
+    triggererRef,
```

Comment:
@saurabhdaware can you check this once, not sure ye kaha se agaya

Reply:
this to i have added  in -
``` packages/blade/src/components/Input/DropdownInputTriggers/types.ts ```